cpp/include/cudf/io/parquet.hpp

```diff
@@ -101,6 +101,8 @@ class parquet_reader_options {
   bool _allow_mismatched_pq_schemas = false;
   // Cast timestamp columns to a specific type
   data_type _timestamp_type{type_id::EMPTY};
+  // Whether to use JIT compilation for filtering
+  bool _use_jit_filter = false;
```

Comment:
Perhaps in a future PR but should use JIT filter `true` by default since it is more performant. Are there any caveats with this?

Reply:
It is still experimental. The caveat is the first-time JIT compilation time, which can be in the order of ms.
Also, the JIT compilation time is dependent on the complexity of the kernel.